dgraphtest/paths.go

```diff
@@ -55,7 +55,7 @@ func init() {
 		panic(err)
 	}
 
-	log.Printf("[INFO] baseRepoDir: %v", baseRepoDir)
-	log.Printf("[INFO] repoDir: %v", repoDir)
-	log.Printf("[INFO] binariesPath: %v", binariesPath)
+	// log.Printf("[INFO] baseRepoDir: %v", baseRepoDir)
```

Comment:
Why are we removing this?

Reply:
It's alright if these gets printed, but maybe we need to do something about the init function